server/v2/types.go

```diff
@@ -0,0 +1,14 @@
+package serverv2
+
+import (
+	"cosmossdk.io/core/transaction"
+	"cosmossdk.io/server/v2/appmanager"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+)
+
+type Application[T transaction.Tx] interface {
+	GetAppManager() *appmanager.AppManager[T]
+	GetConsensusAuthority() string
+	InterfaceRegistry() codectypes.InterfaceRegistry
+	// GetStore() any
```

Comment:
Any reason we don't make runtime return its store but let simapp do it?

Reply:
its will break the interface I think, `simapp.GetStore()` returns as runtimev2.Store that we shouldn't import here. That why I moved Store out as an any field